dspace-jspui/src/main/webapp/search/discovery.jsp

```diff
@@ -486,7 +486,7 @@ else if( qResults != null)
 	
 	if (pageFirst != 1)
 	{
-	    %><li><a href="<%= firstURL %>">1</a></li><li>...</li><%
+	    %><li><a href="<%= firstURL %>">1</a></li><li class="disabled"><span>...<span></li><%
```

Comment:
Just realized that this line should have a closing `span` tag.  So the code should be (`<span>...</span>`)